packages/discovery-react-components/src/components/DocumentPreview/DocumentPreview.tsx

```diff
@@ -163,20 +164,23 @@ function PreviewDocument({
   hideToolbarControls,
   setHideToolbarControls,
   highlight,
+  setCurrentPage,
   fallbackComponent
 }: PreviewDocumentProps): ReactElement | null {
   // if we have PDF data, render that
   // otherwise, render fallback document view
   if (file) {
     return (
-      <PdfViewer
+      <PdfViewerWithHighlight
         file={file}
-        document={document}
+        document={document ?? undefined}
```

Comment:
Why is this necessary?

Reply:
`Props` above defines it as:

```
document?: QueryResult;
```

which is equivalent to `QueryResult || undefined`.

Reply:
It was necessary because the `document` there is `QueryResult |  null | undefined`. I revise the type definition as I agree that `document ?? undefined` looks strange.